conference/src/test/java/org/spine/samples/lobby/conference/ConferenceServiceShould.java

```diff
@@ -212,5 +219,141 @@ public Void apply(Void input) {
                                         .build(), conferenceUpdatedObserver);
     }
 
+    @SuppressWarnings("unchecked")
+    @Test
+    public void publish_conference_and_generate_ConferencePublished_event() {
```

Comment:
This test method is really big. Please extract some methods/fields. Actually, you should have a warning here.